Notifiable-iOS/Network/FWTHTTPSessionManager.m

```diff
@@ -144,12 +146,12 @@ - (void) _buildTaskForPath:(NSString *)path
     NSURLRequest *request = [self _buildRequestWithPath:path method:method andParameters:parameters];
 
     __weak typeof(self) weakSelf = self;
-    [NSURLConnection sendAsynchronousRequest:request queue:self.sessionOperationQueue completionHandler:^(NSURLResponse * _Nullable response, NSData * _Nullable data, NSError * _Nullable connectionError) {
-        NSLog(@"Response with Error: %@", connectionError);
+    NSURLSessionDataTask *task = [self.urlSession dataTaskWithRequest:request completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
+        NSLog(@"Response with Error: %@", error);
```

Comment:
@igorcferreira Can you check if there's an error before logging anything? I'm seeing on a project that this line (on the old version) keeps logging this: `Response with Error: (null)`

Reply:
👍 Will do